src/controllers/controller.bar.js

```diff
@@ -22,14 +22,14 @@ defaults.set('bar', {
 	},
 
 	scales: {
-		x: {
+		_index_: {
```

Comment:
Why the underscore before and after? I'm assuming you want them to be private, but what's the reason for that? We haven't done this with any other options, so I would probably lean towards just "index"

Reply:
Because `index` is not the actual value its going to end up with.  Also to try to not collide with user specified values.